src/main/java/org/sagebionetworks/bridge/spring/controllers/AssessmentConfigController.java

```diff
@@ -37,33 +37,36 @@ final void setAssessmentConfigService(AssessmentConfigService service) {
     public AssessmentConfig getAssessmentConfig(@PathVariable String guid) {
         UserSession session = getAuthenticatedSession();
         String appId = session.getAppId();
+        String ownerId = session.getParticipant().getOrgMembership();
         
-        return service.getAssessmentConfig(appId, guid);
+        return service.getAssessmentConfig(appId, ownerId, guid);
     }
     
     @PostMapping("/v1/assessments/{guid}/config")
     public AssessmentConfig updateAssessmentConfig(@PathVariable String guid) {
         UserSession session = getAuthenticatedSession(DEVELOPER, STUDY_DESIGNER);
         String appId = session.getAppId();
+        String ownerId = session.getParticipant().getOrgMembership();
```

Comment:
For assessment config, also allow developers and admins to manipulate any config (as assessment controller).

Reply:
This has been changed for this controller.